src/widget/woverview.cpp

```diff
@@ -478,6 +506,19 @@ void WOverview::mousePressEvent(QMouseEvent* e) {
     }
 }
 
+void WOverview::keyPressEvent(QKeyEvent* e) {
+    // TODO() this events gets lost on the way. Fix it.
```

Comment:
I don't think we need a TODO for this if right-click works as a cancel method.  But if we do want it, let's say: "TODO: keyPressEvents are not being received by this window during a drag operation. If we want to cancel a drag by ESC we need to figure out why"